macos/ReactTestApp/ReactTestApp.debug.xcconfig

```diff
@@ -6,6 +6,7 @@ GCC_DYNAMIC_NO_PIC = NO
 GCC_OPTIMIZATION_LEVEL = 0
 GCC_PREPROCESSOR_DEFINITIONS = DEBUG=1 $(inherited)
 MTL_ENABLE_DEBUG_INFO = INCLUDE_SOURCE
-ONLY_ACTIVE_ARCH = YES
+ONLY_ACTIVE_ARCH = NO
+ARCHS = $(ARCHS_STANDARD) arm64
```

Comment:
Why do we need to set this? If you're building on an M1, `ARCHS_STANDARD` should already include arm64. If it's only for making sure that things build on CI, can we provide them as parameters to `xcodebuild`?

```
xcodebuild ARCHS="$ARCHS_STANDARD arm64" ONLY_ACTIVE_ARCH=NO ...
```

Reply:
We don't need to. I was trying to ensure that arm64 was built by explicitly stating it. I have confirmed ARCH_STANDARD already has arm64 so we can remove this

Reply:
If it's only for validation, I think it makes more sense to add the variables in [`/.github/workflows/build.yml`](https://github.com/microsoft/react-native-test-app/blob/6f64e208e02fbd09b00201a64cadb70a24262f2c/.github/workflows/build.yml#L273). Adding them here means that devs always have to build both archs even though they only need one.

Reply:
Yeah sorry. I agree that this is better not as an xcconfig. Having this draft PR be public is mostly to iterate and test CI since that seemed the simplest way to go about it. 

Some context for what I was trying to do:
I was trying to make sure FluentUI React Native component static libraries were built as universal binaries. Those are built as a by product of building their test app, which is coming from RNTA. As such, I wanted to make sure RNTA was built into a universal binary (I.E build both architectures). That is done mostly by adding `ONLY_ACTIVE_ARCH=NO` to the xcconfig, since `ARCHS_STANDARD` already includes arm64 with Xcode 12.2

If we would rather ReactTestApp not by default always build both architectures, I could look into changing that on the FluentUI React Native side (Build both architectures separately and combine them). So far I have indeed confirmed that ReactTestApp does compile for arm64 locally, despite it failing CI right now.

Reply:
Talked offline. FluentUI React Native will build a universal binary in it's own pipeline, and this PR will be focused on adding a "Build macOS arm64" step to react-native-test-app's CI